src/index.html

```diff
@@ -10,23 +10,38 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Search-bar</title>
     <link
       rel="stylesheet"
       href="style.css"
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="wrapper">
```

Comment:
Remove unnecessary wrapper, use body instead

Reply:
Not fixed